src/main/java/de/tum/in/www1/artemis/repository/MathExerciseRepository.java

```diff
@@ -0,0 +1,65 @@
+package de.tum.in.www1.artemis.repository;
+
+import static org.springframework.data.jpa.repository.EntityGraph.EntityGraphType.LOAD;
+
+import java.time.ZonedDateTime;
+import java.util.*;
+
+import javax.validation.constraints.NotNull;
+
+import org.springframework.data.jpa.repository.*;
+import org.springframework.data.repository.query.Param;
+import org.springframework.stereotype.Repository;
+
+import de.tum.in.www1.artemis.domain.MathExercise;
+import de.tum.in.www1.artemis.domain.enumeration.AssessmentType;
+import de.tum.in.www1.artemis.web.rest.errors.EntityNotFoundException;
+
+/**
+ * Spring Data JPA repository for the MathExercise entity.
+ */
+@Repository
+public interface MathExerciseRepository extends JpaRepository<MathExercise, Long>, JpaSpecificationExecutor<MathExercise> {
+
+    @Query("""
+                SELECT DISTINCT e FROM MathExercise e
+                LEFT JOIN FETCH e.categories
+                WHERE e.course.id = :#{#courseId}
+            """)
```

Comment:
Multiline Strings should always be aligned with the closing quotation marks
```suggestion
    @Query("""
            SELECT DISTINCT e FROM MathExercise e
            LEFT JOIN FETCH e.categories
            WHERE e.course.id = :#{#courseId}
            """)
```